tortoise/fixture_test.go

```diff
@@ -778,7 +778,7 @@ func TestDisagreement(t *testing.T) {
 		v := new(evotes)
 		if i < n/2 {
 			v = v.support(1, "aa", 0)
-		} else if n == n/2 {
+		} else if i == n/2 {
```

Comment:
@dshulyak This seems like a serious mistake. It passes after the change, but I wonder if the test is correct.

Reply:
i agree thats a mistake, but not that serious :) 

i could have omitted abstain votes alltogether, this test checks that we output `hare|data` but not `valid` if we have hare output, but layer didn't cross tortoise threshold